src/markdown-pages/docs/first-steps/initialising-home-folder.md

```diff
@@ -37,7 +45,7 @@ galasactl local init
 ```
 
 
-This one-time command is run once per user and helps you to quickly create the folder and files that you need. If you already have a Galasa home folder set up, the folder and files are not created. 
+This one-time command is run once Galasa home directory and helps you to quickly create the folder and files that you need. If you already have a Galasa home folder set up, the folder and files are not created. 
```

Comment:
I think there's a word missing? "per/for each Galasa home directory"?

Reply:
oops thank you! :)